src/openrct2/network/network.h

```diff
@@ -42,6 +42,7 @@ extern "C" {
 #endif // __cplusplus
 #include "../common.h"
 #include "../game.h"
+#include "../platform/platform.h"
```

Comment:
I just did a commit to remove all includes of this from header files. Please don't re-introduce it.